python-backend/app/config.py

```diff
@@ -43,7 +43,8 @@ def JWT_ROLE_CALLBACK(jwt_dict):
 
     # Microservice URLs
     DOCUMENT_MS_URL = os.environ.get('DOCUMENT_MS_URL', 'http://localhost:5000')
-    DOCUMENT_MANAGER_URL = os.environ.get('DOCUMENT_MANAGER_URL', 'http://localhost:5000')
+    DOCUMENT_MANAGER_URL = os.environ.get('DOCUMENT_MANAGER_URL',
+                                          'https://document_manager_backend:5001')
```

Comment:
Switch local fallback scheme to http to avoid sorrow in future

Reply:
Yea that was some debugging, good catch.